test/dshell/linker_flag_with_spaces.d

```diff
@@ -0,0 +1,17 @@
+import dshell;
+
+void main()
+{
+    if (OS == "windows" && MODEL == "32")
```

Comment:
I suspect this test is not correct for LDC that uses MODEL=32 for COFF. Maybe just version the test on CRuntime_DigitalMars (assuming the shell uses the same model as the target)?

Reply:
I added an additional `version (DigitalMars)` to restrict it to DMD, while allowing this file to be compiled for Win64 and later invoked with `MODEL=32`, just in case.